frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectSearchAndInbox/index.js

```diff
@@ -1,24 +0,0 @@
-import { Header, InboxSearchComposer, PropertyHouse } from "@egovernments/digit-ui-react-components";
```

Comment:
@prachirane-eGov keep a page for inbox as well , with the same name. Otherwise when I'll merge my code , I will have many conflicts. Render it at /project/inbox

Reply:
Done @nipunarora-eGov

Reply:
Removed unnecessary code from search config @hariomsinha-eGov